OloEditor/src/Panels/NetworkDebugPanel.cpp

```
@@ -0,0 +1,207 @@
+#include "OloEngine/Networking/Core/NetworkManager.h"
+#include "OloEngine/Networking/Core/NetworkSession.h"
+#include "OloEngine/Networking/Core/NetworkLobby.h"
+#include "OloEngine/Networking/Transport/NetworkServer.h"
+#include "OloEngine/Networking/Transport/NetworkClient.h"
+
+#include <imgui.h>
+#include <steam/steamnetworkingsockets.h>
+
+namespace OloEngine
+{
+    void NetworkDebugPanel::OnImGuiRender()
```

Clang (14.0.6)

[warning] method 'OnImGuiRender' can be made static

(readability-convert-member-functions-to-static)